utils/screenshot_utils.py

```
@@ -0,0 +1,405 @@
+            return None
+
+
+async def analyze_screenshot_from_data_url(data_url: str) -> Optional[str]:
+    """分析前端发送的截图DataURL"""
+    try:
+        # DataURL格式: data:image/png;base64,<base64数据>
+        if not data_url.startswith('data:image/'):
+            logger.error(f"无效的DataURL格式: {data_url[:100]}...")
+            return None
+        
+        # 验证DataURL格式，确保包含base64分隔符
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)